add-ons/htpasswd-identity-provider/htpasswd-identity-provider.addon

```diff
@@ -0,0 +1,29 @@
+# Name: htpasswd-identity-provider
+# Description: Configures minishift to use HTPasswdIdentityProvider
+# Url: https://docs.openshift.org/3.9/install_config/configuring_authentication.html#HTPasswdPasswordIdentityProvider
+# Var-Defaults: USERNAME=developer,MINISHFIT_DATA_HOME=/var/lib/minishift,CONFIG_LOCATION=openshift.local.config/master,MASTER_CONFIG_FILE=master-config.yaml
+# Required-Vars: USER_PASSWORD,USERNAME
```

Comment:
You also need to put `CONFIG_LOCATION` in the Required-Vars which then passed from the user input if user wants to try out it with 3.10.